src/components/CheckForm/CheckForm.tsx

```diff
@@ -196,9 +196,9 @@ export const CheckForm = ({ check, disabled, pageTitle }: CheckFormProps) => {
                 <CheckFormAlert />
               </FormLayout.Section>
               <FormLayout.Section label="Execution" fields={[`probes`, `frequency`, ...probesFields]} status={status}>
+                <CheckUsage checkType={checkType} />
                 <CheckProbeOptions checkType={checkType} />
                 {ProbesComponent}
-                <CheckUsage checkType={checkType} />
```

Comment:
We agreed with @majavojnoska we'd keep this at the bottom for now so the workflow made more sense.

Configure probes -> set frequency -> review usage -> submit

If we move it to the top it goes:

Configure probes -> set frequency -> scroll back to the top of the page -> review usage -> scroll to the bottom of the page -> submit